sunpy/time/tests/test_time.py

```diff
@@ -34,11 +34,11 @@ def test_parse_time_microseconds_excess_trailing_zeros():
     assert dt.scale == 'utc'
 
     # Excess digits beyond 6 digits should error if they are not zeros
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match="Input values did not match any of the formats where the format keyword is optional:\n- 'datetime': Input values for datetime class must be datetime objects\n- 'ymdhms': input must be dict or table-like\n- 'iso': Time 2010-Oct-10 00:00:00 does not match iso format\n- 'isot': Time 2010-Oct-10 00:00:00 does not match isot format\n- 'yday': Time 2010-Oct-10 00:00:00 does not match yday format\n- 'datetime64': Input values for datetime64 class must be datetime64 objects\n- 'fits': Time 2010-Oct-10 00:00:00.1234567 does not match fits format\n- 'byear_str': Time 2010-Oct-10 00:00:00.1234567 does not match byear_str format\n- 'jyear_str': Time 2010-Oct-10 00:00:00.1234567 does not match jyear_str format\n- 'astropy_time': Input values for astropy_time class must all be the same astropy Time type."):
```

Comment:
Don't list all of the formats. Just catch the start of the error message.

Reply:
Done. Thank You.